classes/app.rb

```diff
@@ -9,6 +9,7 @@ class App
   def initialize
     @persons = []
     @input = Input.new(self)
+    load_data_from_files
```

Comment:
Kindly fix the error in the image below, that is preventing your application from running.

<img width="396" alt="image" src="https://github.com/sameedmohsin28/OOP-school-library/assets/27709832/3d5e1b3c-0298-483b-9304-99caf6059376">